deybyr647-poem.js

```diff
@@ -0,0 +1,16 @@
+const author = {firstName: "Deyby", lastName: "Rodriguez"}
+
+let message = `This poem is by ${author.firstName} ${author.lastName}. Enjoy`
+
+let poem = ["Roses Are Red", "Violets Are Blue", "I love JavaScript", "I hope you do too!"]
+
+let poemOut = () => {
+  let i;
+  for(i=0; i <= poem.length; i++){
+    console.log(poem[i])
+    }
```

Comment:
Please put the brace at the same indent level as line 9

Reply:
Alright, I'll get to it now